packages/nds-checkbox/README.md

```diff
@@ -0,0 +1,134 @@
+# `@nice-digital/nds-checkbox`
+
+> Checkbox component for the NICE Design System
+
+- [`@nice-digital/nds-checkbox`](#nice-digitalcheckbox) - [Installation](#installation) - [Usage](#usage) - [React](#react) - [Props](#props) - [SCSS](#scss) - [HTML](#html)
+
+## Installation
+
+Install Node, and then:
+
+```sh
+npm i @nice-digital/nds-checkbox --save
+```
+
+## Usage
+
+### React
+
+Import the `Checkbox` component from the package and use within JSX:
+
+```jsx
+import React from "react";
+import { Checkbox } from "@nice-digital/nds-checkbox";
+
+// The mimimum to be supplied for the input to render is a value and a name
+
+<Checkbox value="yes" name="contact-permission" />;
+```
+
+> Note: The React component automatically imports the SCSS, so there's no need to import the SCSS directly yourself.
+
+#### Props
+
+##### value
+
+- Type: `string` (required)
+
+The value for the single input.
+
+```js
+<Checkbox value="yes" name="contact-permission" />
+```
+
+##### name
+
+- Type: `string` (required)
+
+The name attribute on the single input
+
+```js
+<Checkbox label="yes" name="contact-permission" />
+```
+
+##### label
```

Comment:
Do we need a hint as well? E.g. https://design-system.service.gov.uk/components/checkboxes/#checkbox-items-with-hints

Reply:
@ediblecode Shall we do this now or wait until there's a need unless there's one you can think of at the moment? We didn't have anything that used hints on comment collection

Reply:
Nevermind, working on it!